### ISSUE TITLE
"Clear all notifications" button overlay bug

### ISSUE THREAD

**Issue by andrea224** [crdroidandroid/issue_tracker#168]
### Issue type

User interface

### Device

Poco F3 (alioth)

### crDroid version

crDroid 9

### Exact version / Build date

9.3 / 2023-04-09

### Bug description

The button "clear all notifications" in the notification panel appears and get overlapped when not necessary in lock screen and AOD

### Steps to reproduce

wait for a notification (both with screen on or off), go to the lockscreen and here it comes the bug: sometimes it just appears on its own, other times I just open the notification panel without opening the notification or doing anything else and the button just sticks there. 

### Relevant log (logcat/build log)

```shell
Sorry but the log is too long to be fitted here, it won't me save the post with the log copied here
https://hastebin.skyra.pw/odaxoboyur.yaml
```

### Screenshots or videos

![photo_2023-04-12_11-29-06](https://user-images.githubusercontent.com/31412144/231428414-f7eaeb61-389c-456c-9852-8967feda74c4.jpg)
![photo_2023-04-06_11-11-09](https://user-images.githubusercontent.com/31412144/231428420-55c34f21-392d-4b5c-8464-e02aa2d81376.jpg)

### Solution

Just press the button or "play" a bit with the notification panel (open and close) 
Anyway sometimes it goes away on its own and other times I just have to mess a bit for removing it

### Acknowledgements

- [X] I've checked device is officially supported (for device specific reports and not source related).
- [X] I'm running latest version available on crdroid.net for this device
- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by andrea224** [comment]
![Screenshot_20230419-143708_crDroid Home~2](https://user-images.githubusercontent.com/31412144/233077269-345892de-1f06-4185-a415-421542ac9827.png)

Issue still present after dirty flash from build CrDroid 9.3 15.04 to build CrDroid 9.4 18.04

**Comment by gwolf2u** [comment]
this should be fixed in builds that are made after 22.04
do report if any issues still occur
